Formula/ffmpeg.rb

```diff
@@ -1,16 +1,43 @@
 class Ffmpeg < Formula
   desc "Play, record, convert, and stream audio and video"
   homepage "https://ffmpeg.org/"
-  url "https://ffmpeg.org/releases/ffmpeg-3.1.4.tar.bz2"
-  sha256 "7c99df75a4dc12d22c0f1ed11d0acf98cac1f8b5fe7a7434344b167f810bcbfa"
-  head "https://github.com/FFmpeg/FFmpeg.git"
+
+  stable do
+    url "https://ffmpeg.org/releases/ffmpeg-3.1.4.tar.bz2"
+    sha256 "7c99df75a4dc12d22c0f1ed11d0acf98cac1f8b5fe7a7434344b167f810bcbfa"
+
+    option "with-ffplay", "Enable FFplay media player"
+    option "with-openh264", "Enable OpenH264 library (v1.5.0)"
```

Comment:
I could not seem to get the build options to be in scope within the blocks any other way, which I needed to be the case to correctly resolve the dependencies. I may be doing something wrong here, but the only solution I could come up with was this duplication.